service/matching/version_sets.go

```diff
@@ -31,33 +31,33 @@ import (
 	"encoding/base64"
 
 	"go.temporal.io/api/serviceerror"
-	"go.temporal.io/api/taskqueue/v1"
+	taskqueuepb "go.temporal.io/api/taskqueue/v1"
 	"go.temporal.io/api/workflowservice/v1"
-	"go.temporal.io/server/api/persistence/v1"
+	persistencespb "go.temporal.io/server/api/persistence/v1"
```

Comment:
Why not persistencepb?

Reply:
Looks like it's because the import is referring to a `server/api` package.
As mentioned above, I'm loving this convention.

Reply:
Because it's an internal **s**erver protocol buffer package, as opposed to public api

(I didn't make the convention, I'm just enforcing it)